cli/cli_util.h

```diff
@@ -23,7 +23,17 @@ kvdb_storage_info_print(
     const size_t         paramc,
     const char *const *  paramv);
 
+enum kvdb_compact_request {
+    req_status,
```

Comment:
Make these upper case to stand out?

Reply:
Most of our enums do use uppercase.  The one's that don't seem to be the ones I created :)